scheduler.go

```diff
@@ -637,9 +639,7 @@ func (s *Scheduler) Clear() {
 		job.stop()
 	}
 	s.setJobs(make([]*Job, 0))
-	if s.tags != nil {
-		s.TagsUnique()
-	}
+	s.TagsUnique()
```

Comment:
We only want to set the tags to unique if they were originally set on the scheduler. So this should probably still have a if check to say if not false then set to an new empty sync map. Am I missing something with how this map works or anything?

Reply:
Ah, I see, the purpose of this originally was to reinstantiate the tag map to clear out the tags. Now that it is using a `sync.Map`, it would need to go through and delete the items within.

Reply:
Fixed and verified with TDD.